addon/components/swiper-container.js

```diff
@@ -206,19 +204,36 @@ export default Component.extend({
    * @param {Object} swiper - Swiper instance
    */
   _slideChanged(swiper) {
-    let index = this.get('loop')
-      ? $(swiper.slides)
-        .filter('.swiper-slide-active')
-        .attr('data-swiper-slide-index')
-      : swiper.realIndex;
+    let index;
+
+    if (this.get('loop')) {
+      Object.keys(swiper.slides).forEach((key) => {
+        if (swiper.slides[key].classList && swiper.slides[key].classList.contains('swiper-slide-active')) {
+          index = swiper.slides[key].getAttribute('data-swiper-slide-index');
```

Comment:
I believe we can use dataset instead of getAttribute as that is the canonical way of interacting with data attributes: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset and the browser support goes back to IE 11.

Reply:
Okay, it makes sense, will try it out.